### DISCUSSION TITLE
按照你的步署方法，后端完全不会弄，我是网络小白！

### DISCUSSION THREAD

**Discussion by channychan11** [OXeu/Rin#227] [Q&A]
按照你的部署方法，后端完全不会弄，我是网络小白！
后端地址不知道是什么？
部分变量不知道填什么？

RIN_GITHUB_CLIENT_ID=<你的GithubClientID>
RIN_GITHUB_CLIENT_SECRET=<你的GithubClientSecret>
JWT_SECRET=<JWT 认证所需密钥，可为常规格式的任意密码>
S3_ACCESS_KEY_ID=<你的S3AccessKeyID>
S3_SECRET_ACCESS_KEY=<你的S3SecretAccessKey>

以上这些环境参数不知道在那里可以找到

**Comment by OXeu** [comment]
文档末尾有演示视频，跟着演示视频操作即可

**Comment by channychan11** [comment]
好的，谢谢，之前视频打不开，现在可以了。我跟着做试试，如果有什么问题再麻烦你

Xeu ***@***.***> 于2024年7月24日周三 17:38写道：
